MobileCenter/MobileCenter.xcodeproj/xcshareddata/xcschemes/MobileCenter.xcscheme

```diff
@@ -39,6 +39,16 @@
                ReferencedContainer = "container:MobileCenter.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
```

Comment:
Please revert this change. MobileCenter scheme doesn't need to run Crashes Test.